//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acdefg_NO_AP.h
//
// Code generation for function 'acdefg_NO_AP'
//

#ifndef ACDEFG_NO_AP_H
#define ACDEFG_NO_AP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdefg_NO_AP(double P_init, double V_init, double A_init, double P_wayp,
                  double A_wayp, double V_max, double A_min, double J_max,
                  double J_min, creal_T t[14]);

#endif
// End of code generation (acdefg_NO_AP.h)
